docs-chef-io/content/inspec/resources/aws_api_gateway_v2_integration_responses.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "aws_api_gateway_v2_integration_responses Resource"
+platform = "aws"
+draft = false
+gh_repo = "inspec-aws"
+
+[menu.inspec]
+title = "aws_api_gateway_v2_integration_responses"
+identifier = "inspec/resources/aws/aws_api_gateway_v2_integration_responses Resource"
+parent = "inspec/resources/aws"
++++
+
+Use the `aws_api_gateway_v2_integration_responses` InSpec audit resource to test properties of multiple AWS APIGatewayV2 IntegrationResponses.
+
+For additional information, including parameters and properties, see the [AWS documentation on AWS APIGatewayV2 IntegrationResponse](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integrationresponse.html).
+
+## Installation
+
+{{% inspec_aws_install %}}
+
+## Syntax
+
+Ensure that the integration response exists.
+
+```ruby
+describe aws_api_gateway_v2_integration_responses(api_id: 'APP_ID', integration_id: 'INTEGRATION_ID') do
+  it { should exist }
+end
+```
+
+## Parameters
+
+`api_id` _(required)_
+: The API identifier.
+
+`route_id` _(required)_
+: The route identifier.
+
+## Properties
+
+`content_handling_strategies`
+: Supported only for WebSocket APIs. Specifies how to handle response payload content type conversions. Supported values are CONVERT_TO_BINARY and CONVERT_TO_TEXT, with the following behaviors:
+CONVERT_TO_BINARY: Converts a response payload from a Base64-encoded string to the corresponding binary blob.
+CONVERT_TO_TEXT: Converts a response payload from a binary blob to a Base64-encoded string.
+If this property is not defined, the response payload will be passed through from the integration response to the route response or method response without modification.
+
+: **Field**: `content_handling_strategy`
+
+`integration_response_ids`
+: The integration response ID.
+
+: **Field**: `integration_response_id`
+
+`integration_response_keys`
+: The integration response key.
+
+: **Field**: `integration_response_key`
+
+`response_parameters`
+: A key-value map specifying response parameters that are passed to the method response from the backend. The key is a method response header parameter name and the mapped value is an integration response header value, a static value enclosed within a pair of single quotes, or a JSON expression from the integration response body. The mapping key must match the pattern of method.response.header.{name}, where name is a valid and unique header name. The mapped non-static value must match the pattern of integration.response.header.{name} or integration.response.body.{JSON-expression}, where name is a valid and unique response header name and JSON-expression is a valid JSON expression without the $ prefix.
```

Comment:
Here too the name and jsonexpression needs to be curly braces?

Reply:
This is fine

Reply:
ok